src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -267,8 +301,8 @@ Actor* func_80A74674(PlayState* play, Actor* actor) {
     return NULL;
 }
 
-void func_80A74714(EnIk* this) {
-    f32 frames = Animation_GetLastFrame(&object_ik_Anim_00CD70);
+void EnIk_SetupStandUp(EnIk* this) {
+    f32 frames = Animation_GetLastFrame(&gIronKnuckleStandUpAnim);
     f32 frame;
```

Comment:
```suggestion
    f32 endFrame = Animation_GetLastFrame(&gIronKnuckleStandUpAnim);
    f32 startFrame;
```
Looks like there are other variables called frames below that get set as `Animation_GetLastFrame`. Would probably rename those to `endFrame` as well.